symengine/symengine_assert.h

```diff
@@ -17,14 +18,33 @@
             abort();                                                           \
         }                                                                      \
     }
-#else
+#endif // !defined(SYMENGINE_ASSERT)
+
+#if !defined(SYMENGINE_ASSERT_MSG)
+#define SYMENGINE_ASSERT_MSG(cond,msg)                                         \
+    {                                                                          \
+        if (!(cond)) {                                                         \
+            std::cerr << "SYMENGINE_ASSERT failed: " << __FILE__               \
+                      << "\nfunction " << __func__ << "(), line number "       \
+                      << __LINE__ << " at \n"                                  \
+                      << XSTR(cond) << "\n"                                    \
+                      << "ERROR MESSAGE:" << "\n"                              \
```

Comment:
You can probably merge `"ERROR MESSAGE:" << "\n"` into `"ERROR MESSAGE:\n"` to help `clang-format`.

Reply:
Thanks for the tip!